src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -209,8 +209,19 @@ internal static SecurityZone GetFileSecurityZone(string filePath)
         /// </remarks>
         private static SecurityZone MapSecurityZone(string filePath)
         {
+            // WSL introduces a new filesystem path to access the Linux filesystem from Windows, like '\\wsl$\ubuntu'.
+            // If the given file path is such a special case, we consider it's in 'MyComputer' zone.
+            if (filePath.StartsWith(@"\\wsl$", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should `\\wsl$` be added to https://github.com/PowerShell/PowerShell/blob/8901d2a5099f5173373b77d95c5b9934300732e3/src/System.Management.Automation/engine/Utils.cs#L1552 as a constant?

Reply:
Will make the change.